code-examples/quarkus/kafka-quickstart/pom.xml

```diff
@@ -98,12 +97,16 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-openshift</artifactId>
+      <artifactId>quarkus-kubernetes</artifactId>
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-docker</artifactId>
     </dependency>
+    <dependency>
```

Comment:
this should be in a profile as it's not a dependency for all types of build

Reply:
Build container needs to be SBO enabled (we building it now using CI/CD). Profiles are switched at runtime